Makefile

```diff
@@ -105,15 +105,11 @@ vendor:
 # Create YAML file for deployment
 dev-yaml: | $(KUSTOMIZE)
 	@$(KUSTOMIZE) version 2>&1 | grep -q 'KustomizeVersion:v\{0,1\}[2-9][[:digit:]]\{0,\}' || { echo "kustomize v2+ required" 1>&2; exit 1; }
-	@echo "updating kustomize image patch file for manager resource"
-	sed -i'' -e 's@image: .*@image: '"${DEV_IMG}"'@' ./config/default/vsphere_manager_image_patch.yaml
-	cmd/clusterctl/examples/vsphere/generate-yaml.sh
+	VSPHERE_MANAGER_IMG=${DEV_IMG} cmd/clusterctl/examples/vsphere/generate-yaml.sh
```

Comment:
Hi @andrewsykim,

If there's nothing left in the target but this line, then why don't you move the kustomize version check to `generate-yams.sh`?

Reply:
Would you prefer `generate-yaml.sh` also use `hack/.bin/kustomize`?

Reply:
I don't really care. At that point there's no cause to do so. However, if you notice at the top, we're exporting the PATH to prefer `hack/.bin`, so the script will use the specific version if called via the Makefile.

Reply:
Would prefer to get rid of the version check entirely then

Reply:
removed the version check

Reply:
removed the version check

Reply:
That makes sense. Keep the order only dependency to enforce the download. Thanks.